modules/nlu/src/config.ts

```diff
@@ -32,4 +32,18 @@ export interface Config {
    * @default 850mb
    */
   modelCacheSize: string
+
+  /**
+   * Maximum number of concurrent trainings per Botpress instance
+   * @default 2
+   * @optional
+   */
+  maxTrainingPerInstance?: number
+
+  /**
+   * Whether or not to start bots that require training on mount
+   * @default true
+   * @optional
+   */
+  queueTrainingOnBotMount?: boolean
```

Comment:
The description doesn't seem to fit with the name of the configuration

Reply:
good catch toga wrok on me elgnish